src/tests/test_magic.py

```diff
@@ -187,6 +187,11 @@ def test_persist_no_index(ip):
     assert persisted == [(1, "foo"), (2, "bar")]
 
 
+def test_unrecognized_arguments(ip):
+    result = ip.run_cell("%sql --stuff sqlite://")
+    assert "Unrecognized argument(s)" in str(result.error_in_exec)
+
```

Comment:
here's an example that causes this code to fail:

```
In [4]: %load_ext sql
The sql extension is already loaded. To reload it, use:
  %reload_ext sql

In [5]: %sql duckdb://

In [6]: %sql select * from penguins.csv limit --some sql comment
UsageError: Unrecognized argument(s): --some
If you need help solving this issue, send us a message: https://ploomber.io/community

In [7]:

```

we cannot accurately parse --arguments in a line magic because they might be comments

Reply:
please add this as a test, it'll fail but it'll allow us to evaluate the solution

Reply:
Hi @tl3119,
I don't see the test case, Eduardo mentioned, added to `test_unrecognized_arguments_cell_magic`. Please add the test case using `pytest.parametrize`

Reply:
I have added the test case using pytest.parametrize